include/rocksdb/memtablerep.h

```diff
@@ -39,24 +39,34 @@
 #include <stdexcept>
 #include <stdint.h>
 #include <stdlib.h>
+#include <rocksdb/slice.h>
 
 namespace rocksdb {
 
 class Arena;
 class Allocator;
 class LookupKey;
-class Slice;
 class SliceTransform;
 class Logger;
 
 typedef void* KeyHandle;
 
+extern Slice GetLengthPrefixedSlice(const char* data);
```

Comment:
Hopefully for the skiplist compilation the `utils/coding.h` is available and compiler will have a chance to inline this.